app/Http/Controllers/Web/GroupTypesController.php

```diff
@@ -40,6 +40,8 @@ public function store(Request $request)
         $values = $this->validate($request, array_merge_recursive($this->rules, [
             'name' => [Rule::unique('group_types')],
         ]));
+        // @see ActionsController->fillInOmittedCheckboxes
+        $values['filter_by_state'] = $request->has('filter_by_state');
```

Comment:
Now that we're using this in two places, I wonder if it'd be helpful to generalize as a helper?

```php
function withOmittedBooleans(Request $request, $model): Request
{
    $booleans = array_keys(((new $class)->casts), 'boolean');

    foreach ($booleans as $field) {
        $request[$field] = $request->has($field);
    }

    return $request;
}
```

Then this could just read:

```php
$request = withOmittedBooleans($request, GroupType::class);

$this->validate($request, // ...
```

Reply:
That's an awesome idea! Just to clarify, you're thinking this goes in the [app/helpers.php](https://github.com/DoSomething/rogue/blob/master/app/helpers.php) ? Those functions have a mix of camelCase and snake_case

Reply:
I'll add this in a followup PR.

Reply:
Yes! And oh good catch – let's do `with_omitted_booleans`.